react/features/base/conference/middleware.any.ts

```diff
@@ -690,3 +696,33 @@ function _p2pStatusChanged(next: Function, action: AnyAction) {
 
     return result;
 }
+
+/**
+ * Notifies the feature base/conference that the action
+ * {@code SET_ASSUMED_BANDWIDTH_BPS} is being dispatched within a specific
+ *  redux store.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the
+ * specified {@code action} to the specified {@code store}.
+ * @param {Action} action - The redux action {@code SET_ASSUMED_BANDWIDTH_BPS}
+ * which is being dispatched in the specified {@code store}.
+ * @private
+ * @returns {Object} The value returned by {@code next(action)}.
+ */
+function _setAssumedBandwidthBps({ getState }: IStore, next: Function, action: AnyAction) {
+    const state = getState();
+    const conference = getCurrentConference(state);
+    const payload = Number(action.assumedBandwidthBps);
+
+    const assumedBandwidthBps = isNaN(payload) || payload < ASSUMED_BANDWIDTH_BPS
```

Comment:
Can we pl rename `ASSUMED_BANDWIDTH_BPS` to `MIN_ASSUMED_BANDWIDTH_BPS` ? Makes it easier to read.